vhdl_ls/src/stdio_server.rs

```diff
@@ -134,10 +134,10 @@ fn read_request(reader: &mut BufRead) -> String {
 
 fn send_response(writer: &mut Write, response: &str) {
     trace!("SEND RESPONSE: {:?}", response);
-    write!(writer, "Content-Length: {}\r\n", response.len());
-    write!(writer, "\r\n");
+    writeln!(writer, "Content-Length: {}", response.len());
```

Comment:
The protocol requires it to be \r\n, writeln will just insert \n on most platforms.